File: source/frontend/src/app/entities/vacature/postbusadres.ts
import {PostbusadresImpl} from './postbusadresImpl';

export interface Postbusadres {
  postbusadres?: PostbusadresImpl;
}

export function isPostbusadres(object: any): object is Postbusadres {
  return 'postbusadres' in object;
}

File: source/backend-vacatures-bron/src/main/java/com/cap/pocvng/entity/Postbusadres.java
package com.cap.pocvng.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Embedded;
import javax.persistence.Entity;
import javax.validation.Valid;

/**
 * Postbusadres
 */

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Postbusadres extends AdresDetailsNederland {

    @Valid
    @Embedded
    private PostbusadresImpl postbusadres;

}
